lib/advisor/core/answers/answers.ex

```diff
@@ -1,13 +1,30 @@
-defmodule Advisor.Core.AnswerFinder do
-  alias Advisor.Core.Answer
+defmodule Advisor.Core.Answers do
   alias Advisor.Repo
+  alias Advisor.Core.Answer
   import Ecto.Query
 
+  def store(params) do
+    Repo.insert_all(Answer, all_answers(params), returning: true)
+  end
+
+  def all_answers(%{"id" => advice_request_id} = params) do
+    Enum.reduce(params, [], fn({question_id, answer}, acc) ->
```

Comment:
Excellent call. While `acc` is short and known in `FP`, it has not valuable meaning `here`. I'll find a better name for it 👍